tools/vsimporter/xib2nib/XIBObject.cpp

```diff
@@ -275,6 +276,11 @@ void XIBObject::ScanStoryObjects(pugi::xml_node node) {
     _id = getAttrAndHandle("id");
     _className = node.name();
     _isStory = true;
+	
+	if (strstr(_className, "Controller") != NULL)
```

Comment:
why strstr instead of strcmp? 
it's looking for a substring specifically... though i don't think it's specific enough. it should be something like 

if (strcmp(_className, "viewController") == 0)